OneCommunityGlobal/HGNRest
# Imran added getProjectMembers permission

# Description
Gives the user permission to access the profile of any user directly from the projects members page. "Other Links" -> "Projects" -> "Members"

## Related PRS (if any):
To test this backend PR you need to checkout the [#2263](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2263) frontend PR.
…

## Main changes explained:
- updated getprojectMembership function to return _id if requestor has permission

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
6. Follow steps in Frontend PR[#2263](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2263) to test
